projects/docs/docs/commands/test.md

```diff
@@ -46,15 +46,28 @@ tuist test --device "iPhone X" --os 14.0
 One of the benefits of using Tuist over other automation tools is that developers can get familiar with a set of commands that they can use in any Tuist project.
 :::
 
+**Test in parallel**
+
+```bash
+tuist test --additional-parameters -parallel-testing-enabled
```

Comment:
We are not supporting these params yet right?

Reply:
Removed